easybuild/tools/repository/filerepo.py

```diff
@@ -109,7 +109,11 @@ def add_easyconfig(self, cfg, name, version, stats, previous):
         else:
             statstxt = statscomment + statsprefix + stats_to_str(stats, isyeb=yeb_format) + statssuffix
 
-        txt += statstxt
+        try:
+            txt += statstxt
+        except UnicodeDecodeError as e:
+            raise EasyBuildError("Got a unicode error here: type(txt) %s type(statstxt) %s", type(txt), type(statstxt))
```

Comment:
Values should also be printed? (not sure if that's feasible without triggering an exception that caused the problem in the first place though :)